src/modules/evconf_konva/config_stage.ts

```diff
@@ -131,7 +131,8 @@ export default class ConfigStage {
     context.set_container(this._stage.container());
     this.context.add_observer((ev) => this._handle_stage_context_event(ev));
     this.registerListeners();
-    this.resizeStage();
+
+    setTimeout(() => this.resizeStage(), 1500); // we have to wait for the animation of the splitpanes to finish
```

Comment:
Time is not a synchronization primitive.

Can we instead listen to an event of splitpanes? See <https://antoniandre.github.io/splitpanes/#emitted-events>

Reply:
@thexyno I tried, but they don't fire on the initial rescale, only on the later ones. Waiting for time isn't really a problem though, as it is impossible to get into a situation where you notice that the initial rescale hasn't happen before, as you don't have anything on the stage by default. And 1500ms is more than enough for the animation to finish. I know it isn't perfect, I don't like it either, but I didn't find a solution which would have been worth the effort as I don't think this will cause any real problems in reality.